test/jdk/javax/swing/JCheckBox/bug8032667.java

```diff
@@ -20,55 +20,71 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+
 import java.awt.BorderLayout;
 import java.awt.Canvas;
 import java.awt.Dimension;
 import java.awt.Graphics;
 import java.awt.Graphics2D;
 import java.awt.Image;
 import java.awt.image.BufferedImage;
-import javax.swing.JApplet;
+
 import javax.swing.JCheckBox;
 import javax.swing.JComponent;
-import javax.swing.SwingUtilities;
+import javax.swing.JFrame;
 
 /* @test
  * @bug 8032667
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary [macosx] Components cannot be rendered in HiDPI to BufferedImage
- * @run applet/manual=yesno bug8032667.html
+ * @run main/manual bug8032667
```

Comment:
Minor:
May add a blank line after this.

Reply:
Thanks added